src/components/Summary/AddNewScoreRedux/index.js

```diff
@@ -0,0 +1,268 @@
+  hasFeedback,
+  label,
+  ...rest
+}) => {
+  const FormItem = Form.Item;
+  const hasError = meta.touched && meta.invalid;
+  return (
+    <FormItem
+      label={label}
+      validateStatus={hasError ? 'error' : 'success'}
+      hasFeedback={hasFeedback && hasError}
+      help={hasError && meta.error}
+    >
+      <Rate allowHalf {...input} {...rest} children={children} />
+    </FormItem>
+  );
+};
+
+const FieldizeComment = (height, width) => ({
+  input,
+  meta,
+  children,
+  hasFeedback,
+  label,
+  ...rest
+}) => {
+  const FormItem = Form.Item;
+  const hasError = meta.touched && meta.invalid;
+  const heightpx = height.toString() + 'px';
+  const widthpx = width.toString() + 'px';
+  return (
+    <FormItem
+      label={label}
+      validateStatus={hasError ? 'error' : 'success'}
+      hasFeedback={hasFeedback && hasError}
+      help={hasError && meta.error}
+    >
+      <TextArea
+        bordered={false}
+        {...input}
+        {...rest}
+        children={children}
+        style={{
+          height: heightpx,
+          width: widthpx,
+          overflowY: 'auto',
+        }}
+      ></TextArea>
+    </FormItem>
+  );
+};
+const FieldizeInput = (height, width) => ({
+  input,
+  meta,
+  children,
+  hasFeedback,
+  label,
+  ...rest
+}) => {
+  const FormItem = Form.Item;
+  const hasError = meta.touched && meta.invalid;
+  const heightpx = height.toString() + 'px';
+  const widthpx = width.toString() + 'px';
+  return (
+    <FormItem
+      label={label}
+      validateStatus={hasError ? 'error' : 'success'}
+      hasFeedback={hasFeedback && hasError}
+      help={hasError && meta.error}
+    >
+      <Input
+        bordered={false}
+        {...input}
+        {...rest}
+        children={children}
+        style={{
+          height: heightpx,
+          width: widthpx,
+          overflowY: 'auto',
+        }}
+      ></Input>
+    </FormItem>
+  );
+};
+const RateScore = FieldizeRate();
+const CommentScore = FieldizeComment(200, 400);
+const NameScore = FieldizeInput(40, 200);
+//Reference: https://codesandbox.io/s/jzyl70wpk?file=/SimpleForm.js:611-1048
+
+class AddNewScoreFormRedux extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { DisableSubmit: true };
+    this.submitForm = this.props.handleSubmit(async data => {
+      const id = this.props.questionid;
+      const params = {
+        commentRecordId: id,
+        author: data.name,
+        quality: data.quality,
+        hint: data.hint,
+        completeness: data.completeness,
+        tags: 'no more comment',
+        content: data.comment,
+      };
+      await createComment(params);
+      message.success('Add Overall Score successfully');
+      this.props.reset();
+    });
+    this.handleSubmit = event => {
+      event.preventDefault();
+      this.submitForm();
+    };
+  }
+  handleSubmit = event => {
+    event.preventDefault();
+    this.submitForm();
+  };
+
+  onDetectNullValue = e => {
```

Comment:
You could remove this function `onDetectNullValue` since `shouldEnableSubmitButton` has been implemented.